src/App.tsx

```diff
@@ -1,93 +1,18 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
+import { TodoList } from './components/TodoList';
```

Comment:
<img width="1149" alt="Screenshot 2023-08-24 at 10 39 10" src="https://github.com/mate-academy/react_todo-app/assets/53228259/5a913737-fef1-47c6-9f41-2ae5f430f08d">

you didn't push your files

Reply:
if i pass extension i get errors